src/components/views/Login.js

```diff
@@ -42,7 +42,7 @@ however be sure not to clutter your files with an endless amount!
 As a rule of thumb, use one file per component and only add small,
 specific components that belong to the main one in the same file.
  */
-const FormField = props => {
+export const FormField = props => {
   return (
```

Comment:
i would prefere to make a formField js file in ui (like CardButton usw.)
And import Formfield in login and registration